.github/workflows/ci-deploy.yml

```diff
@@ -77,78 +79,29 @@ jobs:
           --tag $ECR_REGISTRY/$ECR_REPOSITORY:$BRANCH \
           --file ./Dockerfile ./
 
-    - name: Kick off Terraform deploy in sysops/ if not Live environment
+    - name: Kick off Terraform deploy in sysops/
       id: sysops-deploy
+      if: ${{ github.event.pull_request.merged }}
```

Comment:
there will never be a pull_request event, this deploy is triggered on push events

Reply:
Ok, I think `if: github.event_name == 'create' && startsWith(github.ref, 'refs/heads')` will work, but it may actually fire too often? (No harm, just an extra deploy of master which is already deployed?)

I wonder if this should be  `...startsWith(github.ref, 'refs/heads/master')`

Reply:
couple things: one, you haven't added a trigger for `create` events to this workflow either so this will never be a true statement. two, if this is for deployments off the main branch then you already have a push event. there's only two possible paths to trigger a deployment and they are distinct.

Reply:
I do wish we had started with a `main` branch on presto...